cypress/e2e/Mock User to Test and Project.cy.ts

```diff
@@ -0,0 +1,37 @@
+describe('Mocking User and Test Project', () => {
+  it('Mock User and Test Project', () => {
```

Comment:
the API stubbing should be moved to a beforeEach block.

Reply:
See answer in the second comment @astanik.